### DISCUSSION TITLE
What is the width of the table of content?

### DISCUSSION THREAD

**Discussion by Fatih20** [squidfunk/mkdocs-material#2904] [Q&A] [answered]
Does it depends on how long the heading in the markdown is? Or is there a fixed max-width where if the content being outlined exceeds that max-width it would just make a new line?

**Comment by squidfunk** [comment, marked as answer]
I'm not sure if I understand what you mean. The width of the table of contents sidebar on the right is fixed at `234px`. If an item in the table of contents doesn't fit, its text breaks onto a new line.

**Reply by Fatih20** [reply]
Yes that's what I'm looking for. Thanks.
